# This message is used for determining the status of the detection API results
bool OKAY # true if valid detections are returned
bool BUSY # true if the detector is busy with another task
bool ERROR # true if an error occurred
